ragoon/executors/cde_embedder.py

```diff
@@ -0,0 +1,68 @@
+import random
+
+from datasets import Dataset
+
+from ragoon.executors.embedder import BaseEmbedder
+from ragoon.models.base import Config
+from ragoon.utils import chromadb_normalize_name
+
+
+class CDEEmbedder(BaseEmbedder):
+    def __init__(self, config: Config):
+        import chromadb
+        from sentence_transformers import SentenceTransformer
+
+        self.model = SentenceTransformer("jxm/cde-small-v1", trust_remote_code=True)
```

Comment:
@gkaretka as there might be further models used in the future, perhaps it might make sense to use the `model` name from the config?